docs/OpenCLCOnVulkan.md

```diff
@@ -887,51 +872,37 @@ These extension built-in functions are not supported:
 
 ### Numerical Compliance
 
-Clspv is not able to reach full accuracy requirements on the supported builtin
-functions in all cases. Instead, currently, it is able to meet the requirements
-tested by OpenCL CTS under relaxed precision requirements. In order to achieve
-this goal, some functions are implemented using the GLSL.std.450 extended or
-core instructions, some are implemented in the builtin library and some are
-emulated by the compiler.
-
-Note: Clspv has been tested against five Vulkan implementations from different
-vendors and is able to achieve the relaxed accuracy requirements broadly.
+Clspv is not able to reach full accuracy requirements on all builtin functions
+using the GLSL.std.450 extended or core instructions or compiler emulation.
+In order to achieve this goal, some functions are implemented using a builtin
+library (from [libclc](https://libclc.llvm.org/)).
 
-#### Implementations Using Core and Extended Instructions
+#### Implementations Using Builtin Library
 
-`add`, `subtract`, `divide`, `multiply`, `assignment`, `not`, `acos`, `asin`,
-`ceil`, `cos`, `cosh`, `exp`, `exp2`, `exp10`, `fabs`, `floor`, `fmax`, `fmin`,
-`log`, `log2`, `log10`, `mad`, `pow`, `powr`, `rint`, `rsqrt`, `sin`, `sinh`,
-`tan`, `trunc`, `half_cos`, `half_exp`, `half_exp2`, `half_exp10`, `half_log`,
-`half_log2`, `half_log10`, `half_powr`, `half_rsqrt`, `half_sin`, `half_tan`,
-`clamp`, `degrees`, `mix`, `radians`, `sign`, `smoothstep`, `step`, `cross`,
-`dot`, `normalize`, `fast_distance`, `fast_length`, `fast_normalize`.
+Find the latest list in clspv's libclc [SOURCES file](https://github.com/llvm/llvm-project/blob/main/libclc/clspv/lib/SOURCES).
 
-#### Implementations Using Builtin Library
+Here is the list as of March 2024:
```

Comment:
Nit/Question: Trying to understand, given there are multiple HW implementations with different caps (and/or use CLVK flags to CLSPV to tell which built-ins they want native vs libclc). Is this list "lowest common denominator" or all those? Trying to understand, how we got this list (given various HW implementations had different caps or its just the two certified).

Reply:
It's just the list of builtins coming from libclc. It's coming from the libclc SOURCES file (the link is above in the PR).